src/ddmd/dclass.d

```diff
@@ -179,6 +179,15 @@ struct ClassFlags
     alias hasDtor = Enum.hasDtor;
 }
 
+enum IS : ubyte
+{
+    com = 1,     // mask for COM class (meaning it derives from IUnknown)
+    cpp = 2,     // mask for a C++ interface
+    objc = 4,    // mask for an Objective-C class/interface
+    _scope = 8,   // mask for a scope class
```

Comment:
The convention is: `keyword_`. Was recently discussed here https://github.com/dlang/dmd/pull/7313#discussion_r150784331